pkg/handler/handler.go

```diff
@@ -262,14 +262,11 @@ func (h handler) unbind(w http.ResponseWriter, r *http.Request, params map[strin
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "invalid binding_uuid"})
 		return
 	}
-
 	planID := r.FormValue("plan_id")
 	if planID == "" {
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "unbind request missing plan_id query parameter"})
 	}
-
 	resp, err := h.broker.Unbind(instanceUUID, bindingUUID, planID)
-
 	if errors.IsNotFound(err) {
 		writeResponse(w, http.StatusGone, resp)
```

Comment:
any reason we deleted the blank lines? they were single lines which isn't egregious and I think it makes it more readable.